client-spark/spark3/src/main/java/org/apache/spark/shuffle/reader/RssShuffleReader.java

```diff
@@ -206,7 +206,7 @@ public Configuration getHadoopConf() {
         CreateShuffleReadClientRequest request = new CreateShuffleReadClientRequest(
             appId, shuffleId, partition, storageType, basePath, indexReadLimit, readBufferSize,
             1, partitionNum, partitionToExpectBlocks.get(partition), taskIdBitmap, shuffleServerInfoList,
-            hadoopConf, dataDistributionType);
+            hadoopConf, dataDistributionType, mapStartIndex, mapEndIndex);
```

Comment:
Could we avoid `mapStartIndex` and `mapEndIndex`? It's the concept of Spark.

Reply:
Got it.

Reply:
> I believe that it will be useful for you.

Could you describe more why this is useful?

Reply:
If taskBitmap can filter more than 50% Blocks, it means that there are skew data.

Reply:
This may be not correct enough.

Reply:
Just give you an idea.

Reply:
This is not precise. 

I think I will introduce a new param to decide whether to use the taskIdsBitmap, which will be specified by external invoker, like spark client. And in spark client, we could use the endMapIndex to analyze the stage is data skew.

Reply:
It's ok for me.